quasar-utils/javac-plugin/src/java11/java/net/corda/plugins/javac/quasar/ASTUtils.java

```diff
@@ -0,0 +1,11 @@
+package net.corda.plugins.javac.quasar;
+
+import com.sun.tools.javac.code.Symbol;
+import com.sun.tools.javac.util.Filter;
+import com.sun.tools.javac.util.Name;
```

Comment:
According to [the documentation](https://docs.oracle.com/en/java/javase/11/docs/api/jdk.compiler/com/sun/tools/javac/package-summary.html), the `com.sun.tools` package is "legacy":
> This package provides a legacy entry point for the javac tool. See the jdk.compiler module for details on replacement APIs.

Using this seems like a Bad Idea :tm:.

Reply:
I agree, unfortunately the jdk.compiler module is not available for java 8. We might decide to only make this work on Corda 5+ but it would be better to have it for all Corda versions

Reply:
You cannot play the "inclusiveness" card while simultaneously _excluding_ all of our CorDapp developers who use Kotlin. But I was actually thinking that this was another reason why using a compiler plugin was the wrong approach in the first place.